final-progII/src/presentation/Screen.java

```diff
@@ -1210,6 +1212,11 @@ public void actionPerformed(ActionEvent arg0) {
 		JButton deleteAbsence__btnNewButton = new JButton("Delete");
 		deleteAbsence__btnNewButton.setBounds(10, 283, 89, 23);
 		infoConsultAbsences__panel.add(deleteAbsence__btnNewButton);
+		
+		idtDelete__textField = new JTextField();
+		idtDelete__textField.setBounds(10, 262, 86, 20);
+		infoConsultAbsences__panel.add(idtDelete__textField);
+		idtDelete__textField.setColumns(10);
```

Comment:
I don't know what is `idtDelete`, It's just a suggestion but maybe you could rename this text field to be more readable.

Reply:
I dont know either. Maybe I'll change it